region/brinstar/red/Alpha Power Bomb Room.json

```diff
@@ -208,6 +208,122 @@
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }}
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
+            "h_canArtificialMorphBombHorizontally",
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 1
+            }},
+            "canTrickyJump",
+            "canHitbox"
+          ]},
+          {"and": [
+            {"ammo": { "type": "PowerBomb", "count": 2}},
+            {"enemyDamage": {
+              "enemy": "Boyon",
+              "type": "contact",
+              "hits": 2
+            }}
+          ]}
+        ]}
+      ],
+      "note": [
+        "Traverse the room with a small number of Power Bombs.",
+        "If the Chozo item is still there, it is best to exit G-Mode before the final Power Bomb goes off to ensure that PLMs aren't overloaded.",
+        "This strat can be done easily with 7 Power Bombs. Simply kill all of the Boyons before progressing.",
+        "The final Power Bomb needs to be placed on the second to last floating platform or later to break the wall.",
+        "Samus can use one Power Bomb and a Boyon hit to get from the door to the second platform, or from the second to fourth platform.",
+        "Alternatively, when next to the Boyons, place a Power Bomb while the Boyon is low, slightly after the peak of the jump, hitbox through the Boyons.",
+        "If the Boyons are killed, Samus can horizontally boost far enough to skip a platform.",
+        "Samus can cross the room with any of the following: 7 PB; 6 PB and a horizontal boost; 5 PB and hitboxing the first Boyons; 5 PB and damage from the first Boyons;",
+        "4 PB and damage from the first Boyons and hitboxing the second; 3 PB and hitting both sets of Boyons.",
+        "Note that if Samus falls into a Samus Eater, she will be teleported into it when exiting G-Mode."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Morph Power Bomb the Blocks (Thorn Damage Boost)",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "canIframeSpikeJump",
+        "canNeutralDamageBoost",
+        "canTrickyJump",
+        {"thornHits": 3},
+        "h_canArtificialMorphPowerBomb",
+        {"samusEaterFrames": 160}
+      ],
+      "note": [
+        "Traverse the room while rolling on the thorns and avoiding the Samus Eaters.",
+        "Roll from the first platform onto the first set of thorns. When just past the first Boyons, turn right to damage boost up and hit the block and fall back on the thorns.",
+        "While still facing right wait for the thorns to hit again; use the boost to get onto the next set of thorns and roll across them into the third Samus Eater before using a Power Bomb.",
```

Comment:
The Samus Eater doesn't grab me when I do this.  Unless I roll back and forth.  And jumping out before the PB goes of isn't terribly hard.

Reply:
Interesting. I'll look into it a bit more and update this

Reply:
I'm not finding a good way to avoid the grab, and its almost always getting me. I'm barely entering and using spring to escape quickly and its still grabbing me.

Reply:
Vileplume grab -> Looks like if you roll in, you won't get grabbed most of the time.  But bouncing in will get grabbed.

2hit-> Hitting the ceiling block is how I always jumped over the 2nd vileplume.  I just noticed you can stop on the safe pixel and turn around after iframes end, but that forces you to take another hit after reaching the next set of spikes plus you will be bouncing more.